src/compass/s1_geocode_slc.py

```diff
@@ -175,10 +175,11 @@ def run(cfg: GeoRunConfig):
         # Save burst corrections and metadata with new h5py File instance
         # because io.Raster things
         with h5py.File(output_hdf5, 'a') as geo_burst_h5:
+            root_path = '/'
             root_group = geo_burst_h5[root_path]
             identity_to_h5group(root_group, burst, cfg)
```

Comment:
```suggestion
            identity_to_h5group(geo_burst_h5, burst, cfg)
```
`h5py.File` object is the same location as `root_group` so no need for it to exist.